src/main/webapp/js/navigation-loader.js

```diff
@@ -59,11 +60,23 @@ function createListItem(childElement) {
  * @param {string} url
  * @param {string} text
  * @return {Element} Anchor element
+ * For navigation links only.
  */
 function createLink(url, text) {
   const linkElement = document.createElement('a');
   linkElement.appendChild(document.createTextNode(text));
   linkElement.href = url;
+  linkElement.classList.add("btn");
+  linkElement.classList.add("btn-light");
+  return linkElement;
+}
+
+// For username message links only
+function createLinkMessage(url, text) {
```

Comment:
If this is only for username message links, it might be clearer to name the method that way ("createUsernameLink" or something)